papas/graphtools/graphbuilder.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, ids, edges):
         '''
         self.ids = ids
         self.edges = edges
-
+        sort_key =  lambda x: x #to match cpp
```

Comment:
It returns a different result if I remove it.
I could change the default sort parameter in DAGFloodfill to sort on object type sort_key =  lambda x: x, but this means sorting will always take place in DAGFloodfill (well unless the user sets the sort back to None). Sorting may not always be needed. Your call.